lang/python/2to3/Makefile

```diff
@@ -0,0 +1,54 @@
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=2to3
+PKG_VERSION:=1.0
+PKG_RELEASE:=1
+
+PKG_MAINTAINER:=Gerald Kerma <gandalf@gk2.net>
+PKG_LICENSE:=GPL-3.0-or-later
+PKG_LICENSE_FILES:=COPYRIGHT
+
+PKG_HOST_ONLY:=1
+BUILDONLY:=1
+HOST_BUILD_DEPENDS:=python3/host
+HOST_PYTHON3_PACKAGE_BUILD_DEPENDS:=lib2to3
```

Comment:
I think this toolz dependency is left over from the 2to3 PyPI package and is no longer needed.

Reply:
removed